packages/cli/src/cli.js

```diff
@@ -0,0 +1,105 @@
+/* @flow */
+
+const program = require('commander');
+// 💩
+const { promisify } = require('util');
+const glob = promisify(require('glob'));
+const { outputFile, readFile, readJson } = require('fs-extra');
+const inquirer = require('inquirer');
+const Promise = require('bluebird');
```

Comment:
y tho?

Reply:
`Promise.each` makes the sequential prompt feature super easy